standards/bulk.md

```diff
@@ -0,0 +1,174 @@
+- Bulk operations **MUST** accept a constrained number of operations in the request body that is indicated in the documentation of the endpoint. By default this value **MAY** be 100 operations, but should be adjusted according to the needs of the endpoint and the entity-size. Requests beyond the limit **MUST** return a `400` response code and standard [error format body](errors.md) similar to:
+```json
+// RESPONSE
+HTTP/1.1 400
+Content-Type: application/problem+json
+{
+    "title": "Invalid Data",
+    "status": 400,
+    "detail": "Operations collection may only contain a maximum of '100' actions per request.",
+    "instance": "/articles",
+    "requestId": "b6d9a290-9f20-465b-bcd3-4a5166eeb3d7"
+}
+```
+
+#### Request
+
+- Bulk operations **MUST** include a request body schema 
+```json
+{
+    "transaction": "ATOMIC" | "ISOLATED",                                   // default: "ATOMIC", optional. transactionality of the request
+    "operations": [                                                         
+        {                                                                   
+            "operationId": "string" | null,                                 // optional consumer generated id to associate with the operation for comparison to result.
+            "action": "CREATE" | "UPDATE" | "CREATE_UPDATE" | "DELETE",     // indicate intent of operation (can use subset, but do not extend)
+            "ifMatch": "string" | null,                                     // if-match is an optional ETag that can be passed for optimistic concurrency
+            "entity": {
+                ... MATCHING ENTITY...                                      // must match entity schema resource from the collection
+            }                                                               // schema on entity is static and not dynamic
+        },
+    ]
+}
+```
+
+- Bulk operation requests **MAY** be designated as `ATOMIC` or `ISOLATED` via the `transaction` field.
+    - The default transaction type **MUST** be `ISOLATED`.
+    - The transaction type **MAY** be optionally left out of the request schema in implementation.
+    - `ATOMIC` transactions will either succeed or fail together.
+    - `ISOLATED` transactions will allow for individual operations to succeed or fail independently.
+- `operationId` **MAY** be used to associate the operation in the request with the resulting operation in the response. This is useful for tracking the outcome of each operation in the response where it might be ambiguous to reference via `entityId`.
+- `action` enumeration **MUST NOT** be extended, but **MAY** be a subset of the enumeration values where not all actions are required.
+- `etag` **MUST** be used supported in the request schema if it is used for other RESTful operations on the same resource.
```

Comment:
Might be a typo here from some renaming?
```suggestion
- `ifMatch` **MUST** be supported in the request schema if ETags are used for other RESTful operations on the same resource.
```

Reply:
totally! thanks.